packages/@ama-sdk/schematics/schematics/code-generator/open-api-cli-generator/open-api-cli.options.ts

```diff
@@ -27,6 +27,14 @@ export type OpenApiCliOptions = CodegenTaskOptions & {
    * @default path to our own custom typescript generator
    */
   generatorCustomPath: string;
+  /**
+   * Comma separated string of options to give to the openapi-generator-cli
+   * Example: debugModels to log the full json structure used to generate models
+   * Example: debugOperations to log the full json structure used to generate operations
+   *
+   * @default ''
+   */
+  globalProperty: string;
```

Comment:
Maybe we should use plural to indicate the multi properties support: `globalProperties`.

Reply:
That's the name of the variable taken by the generator, I agree that the plural would be better but I prefer keep the consistency here